builds.yml

```diff
@@ -273,8 +273,6 @@ env:
   - NODE_DEBUG: ''
   # Used by react-devtools-core
   - EDITOR_URL: ''
-  # Determines if Barad Dur features should be used
-  - BARAD_DUR: ''
```

Comment:
Could you clarify how the "multichain API" being added in this PR differs from what was previously behind this flag? Could we leave the flag in place for now, removing it in a future PR?

This build config file was intended to make it easier to manage functional differences between Flask and stable builds. The idea being that we could enable/disable the flag solely here, and we'd need less churn elsewhere in the codebase for promoting features from "development" to "Flask" to "stable".

Maybe I'm missing some confounding factor in this case though.

Reply:
I'll move this back behind the BARAD_DUR flag and set that env var to be true for the flask build instead

Reply:
ahh sorry, i'll add a `multichain-api` flag to the `features` in the build.yml